blueoil/blueoil_init.py

```diff
@@ -101,6 +102,14 @@
 # ]
 
 
+learning_rate_schedule_map = OrderedDict([
+    ("constant", "'constant' -> constant learning rate."),
+    ("2-step-decay", "'2-step-decay' -> learning rate decrease by 1/10 on {epochs}/2 and {epochs}-1."),
+    ("3-step-decay", "'3-step-decay' -> learning rate decrease by 1/10 on {epochs}/3 and {epochs}*2/3 and {epochs}-1"),
+    ("3-step-decay-with-warmup", "'3-step-decay-with-warmup' -> warmup learning rate 1/1000 in first epoch, then train same as '3-step-decay'"),
```

Comment:
I think `train the same way as '3-step-decay'` sounds more natural than `train same as '3-step-decay'`.

Reply:
@lm-jira thanks

Reply:
@lm-jira I changed message to `train the same way as '3-step-decay'` in 1b82b44